indico/modules/events/registration/legacy_notifications.py

```diff
@@ -0,0 +1,147 @@
+# This file is part of Indico.
```

Comment:
I'm confused, this is not a rename? This is full of legacy crap that's probably gonna be away.